test/integration/endToEnd/authorization.test.js

```diff
@@ -105,3 +106,86 @@ lab.experiment('Authorization', () => {
     })
   })
 })
+
+lab.experiment('Authorization not given to a non existing user', () => {
```

Comment:
`Authorization for missing user`?

Reply:
i'll find a better text and enhance :+1: